packages/sdk-ts/src/client/indexer/grpc/IndexerGrpcArchiverApi.ts

```diff
@@ -305,4 +305,51 @@ export class IndexerGrpcArchiverApi extends BaseGrpcConsumer {
       })
     }
   }
+
+  async fetchDenomHolders({
+    denom,
+    token,
+    limit,
+  }: {
+    denom: string
+    token?: string
```

Comment:
whats the difference between denom and token?
Maybe we should rename this as symbol, at least on the FE side it will be clearer